Usiqee/Module/Global/Fields/CircularImageView/CircularImageView.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  CircularImageView.swift
+//  Usiqee
+//
+//  Created by Amine on 21/04/2021.
+//
+
+import UIKit
+
+class CircularImageView: UIImageView {
```

Comment:
Can be place in another directory instead of `Field`, can be name component, or view